README.md

```diff
@@ -1,120 +1,65 @@
+			A <strong><a href="https://hmn.md/">Human Made</a></strong> project. Maintained by @joehoyle.
+		</td>
+		<td align="center">
+			<img src="https://hmn.md/content/themes/hmnmd/assets/images/hm-logo.svg" width="100" />
+		</td>
+	</tr>
+</table>
 
-A fast image processor to be used with the Tachyon [WordPress plugin](https://github.com/humanmade/tachyon-plugin). node-tachyon can either be run directly in NodeJS on your server, or also utilize AWS Lambda and act as a proxy to the Lambda function.
 
+Tachyon is built with some strong opinions and assumptions:
 
-## Installation
+- Runs on AWS (using CloudFront, Lambda and API Gateway.)
+- Expects original image files to be stored on Amazon S3.
+- Only supports simple image resizing, not a full image manipulation service.
 
-If you're running on OSX make sure you have libvips installed, the easiest way is to use homebrew:
+Tachyon works best with WordPress, coupled with [S3 Uploads](github.com/humanmade/s3-uploads) and the [Tachyon Plugin](https://github.com/humanmade/tachyon-plugin).
 
-```
-brew install homebrew/science/vips --with-webp --with-graphicsmagick
-```
+![](http://joehoyle-captured.s3.amazonaws.com/1EZ7PH9S.png)
 
-Clone and initialise the repo
+---
 
-```
-git clone git@github.com:humanmade/node-tachyon.git
-npm install
-```
+## Installing
 
-## Installation in Production
+Tachyon is simple to install, just use the `cloudformation-template.json` in this repository to create the whole stack using AWS CloudFormation. It will ask you for a few details along the way.
 
-As Tachyon is meant for production use on AWS, we have a fully fledged CloudFormation template that is the easiest way to get a production ready instance of Tachyon.
+You'll need to upload the latest release to Amazon S3 and put the location in the to the CloudFormation stack configuration.
 
-The `cloudformation-template.json` in the root should be used to instantiate a new Stack in CloudFormation. You'll need to build and zip the Lambda function (see below) and place the zip on S3. Pass this ZIP location in the CloudFormation template parameters.
+## Using
 
-## Configuration
+Tachyon provides a simple HTTP interface in the form of:
 
-Populate the `config.json` with the AWS region and bucket name you want to us. In the following format:
+https://{tachyon-domain}/my/image/path/on/s3.png?w=100&h=80
 
-```JSON
-{
-	"region": "eu-west-1",
-	"bucket": "hmn-uploads-eu"
-}
-```
+It's really that simple!
 
-## Running the server
+#### Args Reference
 
-```
-node server.js [port] [--debug]
-```
+| URL Arg | Type | Description |
+|---|----|---|
+|`w`|Number|Max width of the image.|
+|`h`|Number|Max height of the image.|
+|`quality`|Number, 0-100|Image quality.|
+|`crop`|Boolean or "100,50,20,100"|Whether to crop the image, or resize proportionally.|
 
-With no options passed you should see the server running by default on http://localhost:8080
+For more details checkout the [docs](https://engineering.hmn.md/projects/tachyon/).
```

Comment:
This link should be more promenant. Maybe add to the table up the top, or if not, add immediately after the table.